### DISCUSSION TITLE
Raidrive重复上传夸克网盘

### DISCUSSION THREAD

**Discussion by 404gods** [AlistGo/alist#836] [Q&A]
请问大佬，我用alist搭建了夸克网盘，用raidrive通过webdav连接了夸克网盘。我发现用raidrive上传文件到夸克网盘的时候，小文件还正常，稍大的文件(几十M) 每次都自动上传2次，形成2个一样的文件。请问这是咋回事啊？
![image](https://user-images.githubusercontent.com/50573302/160343383-fc690ac7-339a-4523-957e-e272e07333d1.png)

**Comment by 404gods** [comment]
刚才试了几K的小文档上传一次出现4个重复，看来和文件大小无关，应该是webdav不稳定造成的，请大佬费心看看能不能解决

**Comment by 404gods** [comment]
重新安装了V2版本，发现重复上传的情况好像正常了，但是通过raidrive上传的文件，必须要清除一次Alist缓存才能看到，我在配置文件里设置了5分钟清除一次，不会有什么坏处吧

**Comment by xhofe** [comment]
每次上传已经清除了当前目录的缓存，可能只是因为夸克上传文件的延迟。上传完无法及时获取到文件，导致raidrive认为上传失败进行了重试。
